Source/WebKit/WebProcess/Network/webrtc/LibWebRTCNetwork.h

```diff
@@ -92,7 +93,7 @@ class LibWebRTCNetwork : private FunctionDispatcher, private IPC::MessageReceive
     WebRTCMonitor m_webNetworkMonitor;
 #endif
 #if ENABLE(WEB_RTC)
-    WebMDNSRegister m_mdnsRegister;
+    Ref<WebMDNSRegister> m_mdnsRegister;
```

Comment:
Not sure if this is correct since ideally it would not be a `Ref<>` and `WebMDNSRegister` would just forward ref/deref to `LibWebRTCNetwork`. I actually did try to make `LibWebRTCNetwork` ref-counted, but ran into an issue. It's owned by `WebProcess`, so I tried to make `LibWebRTCNetwork` forward its own ref-counting to `WebProcess`. But `WebProcess` isn't ref-counted. Should it be? Or should I just make `LibWebRTCNetwork` inherit from `RefCounted<>`?. What's the best course of action? Or perhaps this patch is fine as is?

Reply:
`WebProcess` is a singleton. You could make it empty refcounted, I guess.

Reply:
Indeed, adding empty ref() / deref() to WebProcess and forward ref-counting.

Reply:
Fixed!